website/docs/cloud-docs/api-docs/stacks.mdx

```diff
@@ -0,0 +1,494 @@
+
+[401]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401
+
+[403]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403
+
+[404]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404
+
+[409]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409
+
+[412]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/412
+
+[422]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422
+
+[429]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/429
+
+[500]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500
+
+[504]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/504
+
+[JSON API document]: /terraform/cloud-docs/api-docs#json-api-documents
+
+[JSON API error object]: http://jsonapi.org/format/#error-objects
+
+# Stacks API
+
+Stacks represent collections of components, replicated across one or more deployments.
+
+## Overview
+
+The scope of the API includes the following endpoints:
+
+| Method | Path | Action |
+| --- | --- | --- |
+| `POST` | `/organizations/:organization_name/stacks` | Call this endpoint to [create a stack](#create-a-stack).|
+| `GET` | `/organizations/:organization_name/stacks` | Call this endpoint to [list existing stacks](#list-stacks).|
+| `GET` | `/stacks/:stack_id` | Call this endpoint to [show a stack](#show-a-stack).|
+| `PATCH` | `/stacks/:stack_id` | Call this endpoint to [update a stack](#update-a-stack).|
+| `DELETE` | `/stacks/:stack_id` | Call this endpoint to [delete a stack](#delete-stack).|
+
+## Create a stack
+
+This endpoint creates a stack for an organization.
+
+`POST /organizations/:organization_name/stacks`
+
+-> **Note:** Stack create access is restricted to users who maintain the specified project, the owners team, teams with the "Manage all projects" permission, and the [organization API token](/terraform/cloud-docs/users-teams-organizations/api-tokens#organization-api-tokens).
+
+### Request Body
+
+This POST endpoint requires a JSON object with the following properties as a request payload.
+
+Properties without a default value are required.
+
+| Key path                                        | Type    | Default   | Description |
+| --- | --- | --- | --- |
+| `data.attributes.name` | string | | The name of the stack, which can only include letters, numbers, `-`, and `_`. This will be used as an identifier and must be unique in the project. |
+| `data.attributes.description` | string | (nothing) | A description for the stack. |
+| `data.attributes.speculative-enabled`| boolean | `false`      | Whether this stack allows automatic [speculative plans][]. Setting this to `false` prevents HCP Terraform from running plans on pull requests, which can improve security if the VCS repository is public or includes untrusted contributors. |
+| `data.attributes.locked` | boolean | `false`| Whether this stack is locked. A lock prevents HCP Terraform from initiating any new runs that would modify the infrastructure managed by the stack. |
```

Comment:
question: Is `runs` the right terminology to use here? Should it be a `stack runs` instead or something else?

Reply:
I would not.  Deployments?